src/components/app-header/styled.ts

```diff
@@ -97,19 +97,20 @@ export const SignInButton = styled.a<{ isDisabled: boolean }>`
   height: 40px;
   font-weight: bold;
   font-size: 12px;
-  color: #FFF;
   flex-flow: row nowrap;
   align-items: center;
-  text-decoration: none;
   user-select: none;
   display: flex;
   text-transform: uppercase;
   justify-content: center;
+  &, &:hover {
+    color: #FFF;
```

Comment:
I guess we can use the bright theme color here

Reply:
done